packages/peregrine/lib/talons/OrderHistoryPage/orderHistoryContext.js

```diff
@@ -0,0 +1,31 @@
+import React, { createContext, useContext, useMemo } from 'react';
+
+import DEFAULT_OPERATIONS from './orderHistoryContext.gql';
+import { useQuery } from '@apollo/client';
+
+const OrderHistoryContext = createContext();
+
+export const OrderHistoryContextProvider = props => {
+    const operations = Object.assign({}, DEFAULT_OPERATIONS, props.operations);
```

Comment:
Will fix this once #2794 lands

Reply:
@tjwiebell it's merged

Reply:
Updated in 6d3da3c. I really wish I could figure out jsdoc generics/templates so Intellisense could understand this merge utility 😭